.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jenkinsci/strict-crumb-issuer-plugin-developers
```

Comment:
I'm confused because teams seems to exists : https://github.com/orgs/jenkinsci/teams/strict-crumb-issuer-plugin-developers. Perhaps doesn't have correct access to repo

Reply:
I imagine that team was manually created :)
Modified to be have admin permission like other teams.